src/js/components/aboutTheData/AgencyDetailsPage.jsx

```diff
@@ -121,7 +120,7 @@ const AgencyDetailsPage = () => {
                                         <h5>Agency Contact Information</h5>
                                         <div className="more-info-note">Contact this Agency with questions about their submissions</div>
                                         <div className="agency-info__website">
-                                            <ExternalLink url={agencyOverview.website} />
+                                            <a target="_blank" rel="noopener noreferrer" href={agencyOverview.website} >{agencyOverview.website}</a>
```

Comment:
I'm not sure about this one either.  Could you ask Jessica if the link to the agency overview is always a government link?  You might need to send a screenshot of the site showing where this link is.

Reply:
should all be govermental links
[@Jessica Tangitau for ticket number 8095 removing the External Link component for gov sites, would the links to the agency overview always be a government link?](https://data-transparency-bfs.slack.com/archives/C0149JM6A82/p1675801125099579)